### PR TITLE
Rs 2022 08 28.1

### PR DESCRIPTION
**Pull request by rschlaikjer**
2022-08-28-231733_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 1]
```diff
@@ -337,39 +351,40 @@ It seems like this time she cooperated since your interests aligned, but you mus
...
 -- 先輩の言葉に、俺は素直に頷けなかった。
-I cannot so readily accept Senpai's words.
+I can't accept Senpai's words that easily.
 }
 [sha:f937e967c0fe5f0428654e695bba71b508229c08]{
 -- Page 216, Offset 15314.
 -- きっと先輩が正しい。アルクェイドが吸血鬼である事はどうしようもない事実だ。
-// even if added her to flow better with the previous line.
-Even if, in all likelihood, she's right. Arcueid being a vampire is an undeniable fact.
+There's a good chance what she's saying is true.
```
Comment (yk1512) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:359]: I don't really like this change, it feels like his level of belief got reduced too much. I read it as more like "I'm sure she's right (but I don't wanna believe it)"

**Review by yk1512 — APPROVED** [review]

**Review by Petrikow — COMMENTED** [review]
eadas

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 2]
```diff
@@ -97,22 +99,22 @@ So...
...
 -- Page 195, Offset 15268.
 -- いやだ、意味がない、とにべもなく断られたのだ。
-She bluntly refused, saying it was pointless, and how she didn't want to.
+She bluntly refused, saying it was pointless, and that she didn't want to.
 }
 [sha:6510ec2d7c8e3d3eb9dc62594408127f8158eae9]{
 -- Page 196, Offset 15269.
 -- ……むむ。あの時のあいつの冷血ぶりを思い返すと、なんかイラッとしてきたぞ……。
-Mmm... Remembering how unfeeling she was back then kind of irritates me.
+%{i}Grr%{/i}... Remembering how heartless she was back then kind of irritates me.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:117]: is the "grr" va driven?

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 3]
```diff
@@ -179,8 +182,8 @@ Arcueid promised to eliminate Vlov as part of our agreement.
 [sha:39b212364ff8dcd155dbed4bfa503f41221ccefe]{
 -- Page 204, Offset 15282.
 -- あいつが運んでくれたからあの陥没現場まで行けたのであって、俺ひとりだったら警察のバリケードを通り抜けられなかっただろう。
-I only managed to show up at the site of the collapse because she carried me there.
- I most likely wouldn't have been able to make it past the police barricades on my own.
+The only reason I even managed to make it to the site of the collapse is because she carried me there.
+ If I'd been alone I probably wouldn't have been able to get past the police barricade.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:186]: alone, I probably

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 4]
```diff
@@ -246,28 +248,34 @@ She's kind of famous to us people, after all."
...
 -- Page 209, Offset 15296.
 -- 先輩は“敵同士である”事は否定しなかった。
-Senpai did not deny the part about them being mutual enemies.
+Senpai didn't deny the part about them being mutual enemies.
 }
 [sha:0197f4b03c8b58ba0547357064458a6468eb8d13]{
 -- Page 210, Offset 15297.
 -- 「話は分かりました。大部分、予想通りの経緯です。
-"I understand what happened now. Most of yours circumstances were as I expected.
+"I understand what happened now. For the most part, it is as I'd assumed.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:261]: MORE contractions!
> For the most part, it's as I'd assumed.

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 5]
```diff
@@ -380,93 +395,95 @@ Senpai even offers me her sympathies.
...
 -- 遠野くん、彼女にどれくらいの事を聞いてるんですか？」
-Tohno-kun, how much did you hear from her?"
+Tohno-kun, just how much did she tell you?"
 }
 [sha:81519c39c403783057947558f260528c34d4711a]{
 -- Page 222, Offset 15326.
 -- 「どのくらいの事って……この街に人間を食い物にする吸血鬼がいて、それを追いかけて来たって事ぐらいだけど」
-How much did I hear? Only that there are vampires in this city who prey on humans, and that she came chasing after them.
+"How much did she tell me...?
+ Well, just that there was a vampire in this city that preys on humans, and how she came here chasing after them."
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:425]: 'after them' doesn't read perfectly, maybe 'after it'? Not to belittle the personhood of vampires of course

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 6]
```diff
@@ -380,93 +395,95 @@ Senpai even offers me her sympathies.
...
 }
 [sha:fcec250ccef7a2e975fcd50e1439b2f050ae5921]{
 -- Page 223, Offset 15327.
 -- 「つまり彼女本人の事も、彼女が追いかけている敵の事も聞かされていないんですね」
-"So basically, she didn't tell you about herself, or the enemy she's chasing, am I right?"
+"Meaning, she didn't tell you anything about herself, or the enemy she's chasing, am I right?"
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:430]: leaning towards dropping comma on meaning

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 7]
```diff
@@ -380,93 +395,95 @@ Senpai even offers me her sympathies.
...
 -- Page 224, Offset 15330.
 -- 「……これ以上って、もう巻きこまれた後なんですけど、俺」
-"More? I've already been involved plenty, thank you very much."
+"Any more people...? %{i}I'm%{/i} already plenty involved, thank you very much."
 }
 [sha:97d471694fd42172de695e0528294780c88ca904]{
 -- Page 224, Offset 15331.
 -- なので部外者扱いはやめてほしい。俺だって立派な当事者だ。
-Meaning, I don't want to be treated like this doesn't concern me――it very much does.
+So please stop treating me like I'm not――I'm as much a part of this as anyone.
```
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:451]: reads fine to me

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 8]
```diff
@@ -488,44 +505,46 @@ It takes time to restore a city infected by vampires back to normal."
...
+// LintConsistency:15630
+Vlov might be dead, but that doesn't mean his minions got wiped out with him.
 }
 [sha:4d0ea22a3ea085e2c59e190f8e9c022be8bf835f]{
 -- Page 229, Offset 15344.
 -- 先輩は街に残った吸血鬼の残党を倒してまわっているのか。
-// almost identical to Offset 15631 in CIEL07_2_2
-Senpai and her people still have the work of defeating the remaining vampires left to them.
+// Almost identical to Offset 15631 in CIEL07_2_2
+Senpai still has to go around the city rooting out any remaining vampires.
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:520]: no lint consistency flag?

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt [hunk 9]
```diff
@@ -488,44 +505,46 @@ It takes time to restore a city infected by vampires back to normal."
...
-"Quite so. This takes steady effort, so it's not something that can be finished in a day or two."
+"Well, that's how it is.
+ It's a slow, methodical process, so it's not the sort of thing that can be wrapped up in just a day or two.
 }
 [sha:8958ef6914302041bf77c6d81f520756a3054b94]{
 -- Page 230, Offset 15346.
 -- そもそも、わたしはここの学生ですし。仕事が終わっても、よっぽどの事がないかぎり街を離れませんよ」
-And besides, I'm a student of this school. Even once my work is over, I'm not leaving the city, excepting that there are outstanding circumstances.
+And besides, I'm a student at this school.
+ Barring any unforeseen circumstances, I'm not leaving this city even after my work is over."
```
Comment (Avuxo) [inline script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:532]: city, even

**Review by Avuxo — APPROVED** [review]
I see...
pretty funny that you hit this file at the same time as Manolo doing basically pt. 2 of this conversation in d9

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt (hunk 2 above)
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:117]: Not sure what I was thinking here...

File: script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt (hunk 8 above)
Comment (rschlaikjer) [reply script/Ciel/Day 7/07_01_CIEL07_2_1MERGE.txt:520]: No, it's not identical - 先輩 is swapped with 先生

**Pull request merged**